docs/UserGuide.md

```diff
@@ -97,6 +97,12 @@ This section serves to help you better understand the terminologies used in this
 4. Rotational Cipher
    1. `rotencode` - Rotates alphabetical characters by a specified integer offset
 
+| Module         | Description                                                   | Parameters                                                                                              |
+| -------------- | ------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------- |
+| `base64encode` | Encodes the data using base64 format                          | N.A                                                                                                     |
+| `rotencode`    | Rotates alphabetical characters by a specified integer offset | 1. `offset`: The rotation offset.<br><br>Must be between -2147483648 (MIN_INT) and 2147483647 (MAX_INT) |
```

Comment:
I think here instead of saying `offset`, I think we should state that is the `moduleArgument` or somewhere so that they know we are referring to `moduleArgument`.